crates/ruff_linter/src/rules/ruff/rules/incorrectly_parenthesized_tuple_in_getitem.rs

```diff
@@ -0,0 +1,83 @@
+// In case we want to change the boolean setting for
+// this rule to an enum, this will make the code change
+// just a little simpler.
+#![allow(clippy::match_bool)]
```

Comment:
I would prefer to use an `if...else` over a match instead of suppressing the clippy warning.

Reply:
Agreed. Sometimes in this kind of situation it can make sense to introduce a custom enum and match on that instead, e.g.

```rs
enum ParenthesesPreference {
    Keep,
    Remove,
}
```

but here I'm not sure I really see the need; I think I'd just stick with `bool`

Reply:
It looks like this has been marked as resolved, but the comment hasn't been addressed